python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -118,6 +118,8 @@ class MimicExplainer(BlackBoxExplainer):
     :type shap_values_output: interpret_community.common.constants.ShapValuesOutput
```

Comment:
It seems like if this is something worth having for mimic_explainer it's worth having in other explainers as well?

Reply:
totally agree, todo for a future PR